includes/Checker/Checks/Plugin_Repo/Trademarks_Check.php

```diff
@@ -146,6 +147,18 @@ class Trademarks_Check extends Abstract_File_Check {
 		'you-tube-',
 	);
 
+	/**
+	 * Lists of trademarks that are allowed but shows in acronym.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @var string[]
+	 */
+	const FOR_USE_ACRONYM = array(
```

Comment:
Maybe rename to `ALLOWED_ACRONYMS`? Makes it more clear.

Reply:
ok